### PR TITLE
Support RFC 268 testing style

### PR DESCRIPTION
**Pull request by SergeAstapov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,9 +128,32 @@ wondering why your feature is not working.
...
+    await click('a.home');
+
+    assert.equal(currentRoute(), 'new.homepage', 'Should be on the new homepage');
+  });
+});
+```
+
+#### `withFeature`
+
+Older pre- RFC232 tests can utilize `withFeature` test helper to turn on a feature for the test.
```
Comment (kategengler) [inline README.md]: It was actually RFC268. But let's refer to them as "new" and "old" -style acceptance tests in the readme

File: tests/acceptance/feature-flags-rfc-232-test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+import { visit, click } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { enableFeature } from 'ember-feature-flags/test-support';
+
+module('Acceptance | feature flags with RFC 232 test format', function(hooks) {
```
Comment (kategengler) [inline tests/acceptance/feature-flags-rfc-232-test.js]: Maybe name file & module "feature flags with new-style acceptance tests"

**Comment by SergeAstapov** [comment]
@kategengler thank you for the feedback!
I was confused which part belongs to which RFC so updated PR title, commit message and test title/file name.

**Comment by kategengler** [comment]
Thanks!

**Pull request merged**